src/infrastructure/figma/transformers/figma-node-transformer.test.ts

```diff
@@ -72,10 +80,15 @@ describe('tryTransformNodeToAtlassianDesign', () => {
 			lastModified: irrelevantLastModified,
 		});
 
+		const fileMetaResponse = generateGetFileMetaResponse({
```

Comment:
**nitpick:** Please, see https://github.com/atlassian-labs/figma-for-jira/pull/229/files#r1585903139.

We don't really need to generate `lastModifiedBy` value here as we are not interested in a specific value for this test -- any value is okay.

Therefore, consider using just this:

```ts
const fileMetaResponse = generateGetFileMetaResponse();

// ...
const result = tryTransformNodeToAtlassianDesign({
  fileKey,
  nodeId: node.id,
  fileResponse,
  fileMetaResponse,
});

// ...

expect(result).toStrictEqual({
  // ..
  lastUpdatedBy: { id: fileMetaResponse.lastModifiedBy.id },
});
```